app/lab/index.html

```diff
@@ -7,7 +7,7 @@
     <link
       id="jupyter-lite-main"
       rel="preload"
-      href="./build/bundle.js"
+      href="../build/bundle-lab.js"
```

Comment:
Still getting my head aroudn stuff, but is there any way `bundle-<app>.js` could end up in a more atomically-removable place, e.g.:
- in `./lab/bundle.js`
- in `../build/lab/bundle.js`

Reply:
i wonder if this is also good time to add cache-busting prefix to the name of this file... it will of course complicate a lot of things, as we won't _know_ the name of the file until it builds. perhaps we can get webpack to always emit something (e.g. stats) that we can track in `dodo` (but not ship).

Reply:
or maybe we can somehow template `bundle.js?x=<sha256sum>` into the html files...

Reply:
> or maybe we can somehow template `bundle.js?x=<sha256sum>` into the html files...

Agree we should start looking into that at some point. But maybe this can be deferred to a follow-up PR?

Reply:
sure!